tests/test_param_routing.py

```diff
@@ -64,6 +67,16 @@ def test_routing_kwargs(wrapper, builder):
     X = np.random.uniform(size=(n, d)).astype(float)
     y = np.random.choice(n_classes, size=n).astype(int)
 
+    # build once to get expected meta-parameters
+    expected_meta = (
+        wrapper(model=builder, model__hidden_layer_sizes=(100,)).fit(X, y)._meta
```

Comment:
:-1:

Reply:
Idem above. Swapping out for hardcoded values.